src/game/skybox.c

```diff
@@ -61,7 +61,7 @@ struct Skybox {
 
 struct Skybox sSkyBoxInfo[2];
 
-typedef const Texture *const SkyboxTexture[80 * sqr(SKYBOX_SIZE)];
+typedef const Texture *const SkyboxTexture[80 * SKYBOX_SIZE * SKYBOX_SIZE];
```

Comment:
Eh, I think `sqr` looks cleaner here personally

Reply:
Slightly dubious about doing it that way for reasons like the above, though as long as it isn't slower than before in all cases (does use with constants get optimized out?), then I'll let it slide.

Reply:
it should always be faster than before